packages/botbuilder-adapter-slack/src/slack_adapter.ts

```diff
@@ -280,12 +285,19 @@ export class SlackAdapter extends BotAdapter {
      */
     public async validateOauthCode(code: string): Promise<any> {
         const slack = new WebClient();
-        const results = await slack.oauth.access({
+        const details = {
             code: code,
             client_id: this.options.clientId,
             client_secret: this.options.clientSecret,
             redirect_uri: this.options.redirectUri
-        });
+        } 
+        let results = {};
+        if (this.options.oauthVersion == 'v2'|'V2'){
```

Comment:
I don't think this works.

this.options.oauthVersion === 'v2' || this.options.oauthVersion === 'V2'

Reply:
Do we really want to support both v1 and v2? All V1 will have to migrate to v2.

Reply:
YES, we need to support v1 until it is 100% disabled by Slack.

Reply:
```suggestion
        if (this.options.oauthVersion == 'v2'||'V2'){
```
this should suffice, good catch